### PR TITLE
OSDOCS-4838: Document network policy requirements for Web Terminal Operator

### PR DESCRIPTION
**Pull request by amisevsk**
Version(s):
4.10, 4.11, 4.12, main

Issue:
https://issues.redhat.com/browse/OSDOCS-4838

Link to docs preview: https://54890--docspreview.netlify.app/

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by ibuziuk — APPROVED** [review]

File: modules/odc-using-web-terminal.adoc [hunk 1]
```diff
@@ -29,3 +29,8 @@ After the web terminal is initialized, you can use the preinstalled CLI tools li
 * The web terminal remains open until you close it or you close the browser window or tab.
 ====
 
+== Web terminal and network policies
+
+Web terminals may fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the Web Terminal Operator must communicate with the web terminal's pod to verify it is running, and the OpenShift console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal will fail to initialize and the terminal panel will appear to be in a "loading" state.
```
Comment (mburke5678) [inline modules/odc-using-web-terminal.adoc:34]: s/may/might
s/"loading"/`loading`

**Comment by mburke5678** [comment]
@amisevsk A couple of nits. Otherwise LGTM

**Comment by amisevsk** [comment]
@mburke5678 thanks for the review -- made the requested changes.

File: modules/odc-using-web-terminal.adoc [hunk 2]
```diff
@@ -29,3 +29,8 @@ After the web terminal is initialized, you can use the preinstalled CLI tools li
 * The web terminal remains open until you close it or you close the browser window or tab.
 ====
 
+== Web terminal and network policies
+
+Web terminals might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the Web Terminal Operator must communicate with the web terminal's pod to verify it is running, and the OpenShift console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal will fail to initialize and the terminal panel will appear to be in a loading state.
```
Comment (snarayan-redhat) [inline modules/odc-using-web-terminal.adoc:34]:
```suggestion
Web terminals might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the Web Terminal Operator must communicate with the web terminal's pod to verify it is running, and the OpenShift console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize, and the terminal panel appears to be in a loading state.
```

**Review by snarayan-redhat — COMMENTED** [review]
Just one correction to avoid future tense. Rest looks good.

File: modules/odc-using-web-terminal.adoc (hunk 2 above)
Comment (amisevsk) [reply modules/odc-using-web-terminal.adoc]: Updated.

File: modules/odc-using-web-terminal.adoc [hunk 3]
```diff
@@ -29,3 +29,8 @@ After the web terminal is initialized, you can use the preinstalled CLI tools li
 * The web terminal remains open until you close it or you close the browser window or tab.
 ====
 
+== Web terminal and network policies
```
Comment (jboxman-rh) [inline modules/odc-using-web-terminal.adoc:32]: This is missing an `id` for this heading in the form of `[id="web-terminal-and-network-policies_{context}"]`

File: modules/odc-using-web-terminal.adoc [hunk 4]
```diff
@@ -29,3 +29,8 @@ After the web terminal is initialized, you can use the preinstalled CLI tools li
...
+
+Web terminals might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the Web Terminal Operator must communicate with the web terminal's pod to verify it is running, and the OpenShift console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state.
```
Comment (jboxman-rh) [inline modules/odc-using-web-terminal.adoc:34]:
```suggestion
Web terminals might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the Web Terminal Operator must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state.
```

**Review by jboxman-rh — COMMENTED** [review]
@amisevsk, left some comments, thanks!

**Pull request merged**